core/src/test/scala/unit/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -83,8 +85,9 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
     // create adminClient
     val props = new Properties()
     props.put(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG, brokerList)
+    brokerOptions = Array("--bootstrap-server", brokerList)
```

Comment:
Why do we need this now?

Reply:
Oops! Sorry, it should be removed! Thanks.